### ISSUE TITLE
[Accessibility] Focus issue after the three-dot menu

### ISSUE THREAD

**Issue by emrahcom** [jitsi/jitsi-meet#17365]
### What happened?

Our company had an accessibility scan done for Jitsi. This is the description of one problem in the accessibility report:

> When the three-dot menu is opened, the focus is not automatically moved inside of it. Instead, the user has to tab through the whole conference menu to get to the opened menu. The focus should be set into the menu when it is opened and on the triggering element, when it is closed.

### Platform

- [x] Chrome (or Chromium based)
- [ ] Firefox
- [ ] Safari
- [ ] Other desktop browser
- [ ] Android browser
- [ ] iOS browser
- [ ] Electron app
- [ ] Android mobile app
- [ ] iOS mobile app
- [ ] Custom app using a mobile SDK

### Browser / app / sdk version

Any browser

### Relevant log output

```shell

```

### Reproducibility

- [x] The problem is reproducible on meet.jit.si

### More details?

<img width="304" height="214" alt="Image" src="https://github.com/user-attachments/assets/b03e5a3c-b245-4485-bff5-4855b6ad73e1" />
